BankingSystem/BankingSystem/Program.cs

```diff
@@ -0,0 +1,380 @@
+
+            _accountsAdmin.CashTransferOut(fromAccount, toAccount, nameOfTransfer, howManyCashwantTranfer, today);
+
+        }
+
+        void ShowAllOwnAccounts()
+        {
+            Console.WriteLine();
+            Console.WriteLine("\t All available accounts:");
+            Console.WriteLine("-----------------------------");
+            _accountsAdmin.ShowAllAccountsInList();
+            Console.WriteLine();
+        }
+
+        void TransferHistory()
+        {
+            Console.WriteLine();
+            Console.WriteLine("\t Transfer History:");
+            Console.WriteLine("-----------------------------");
+            _accountsAdmin.ShowHistoryTransfer();
+        }
+
+        void Exit()
+        {
+            Console.WriteLine("End program.");
+            Environment.Exit(0);
+        }
+
+
+        string GetUserInputBasedOnMessage(string messageForUser)
+        {
+            string input = "";
+
+            do
+            {
+                Console.WriteLine(messageForUser);
+                input = Console.ReadLine();
+
+                if (string.IsNullOrEmpty(input))
+                {
+                    Console.WriteLine();
+                    Console.WriteLine("!!!!!!!!!!!!!!!!!");
+                    Console.WriteLine("Input was empty");
+                    Console.WriteLine("!!!!!!!!!!!!!!!!!");
+                }
+            }
+            while (string.IsNullOrEmpty(input));
+
+            return input;
+        }
+
+        int GetIntFromUser(string getNumberMessage)
+        {
+            int userNumber = 0;
+            bool parseSuccess = false;
+
+            do
+            {
+                string userIntOption = GetUserInputBasedOnMessage(getNumberMessage);
+                parseSuccess = int.TryParse(userIntOption, out userNumber);
+
+                if (parseSuccess == false)
+                {
+                    Console.WriteLine();
+                    Console.WriteLine("!!!!!!!!!!!!!!!!!");
+                    Console.WriteLine("This is not int:" + userIntOption);
+                    Console.WriteLine("!!!!!!!!!!!!!!!!!");
+                }
+
+            } while (!parseSuccess);
+
+            return userNumber;
+
+        }
+
+        float GetFloatFromUser(string getNumberMessage)
+        {
+            float userFloatNumber = 0;
+            bool parseSuccess = false;
+
+            do
+            {
+                string userFloatOption = GetUserInputBasedOnMessage(getNumberMessage);
+
+                userFloatOption = userFloatOption.Replace(",", ".");
+
+                parseSuccess = float.TryParse(userFloatOption, out userFloatNumber);
+
+                if (parseSuccess == false)
+                {
+                    Console.WriteLine();
+                    Console.WriteLine("!!!!!!!!!!!!!!!!!");
+                    Console.WriteLine("This is not float:" + userFloatOption);
+                    Console.WriteLine("!!!!!!!!!!!!!!!!!");
```

Comment:
You use this approach of showing errors everywere. It's a good candidate for a separate method.